packages/augur-sdk/src/connector/ww-connector.ts

```diff
@@ -1,35 +1,14 @@
 import RunWorker from "../state//Sync.worker";
 
-import { API } from "../state/getter/API";
-import { Augur } from "../Augur";
-import { BlockAndLogStreamerListener } from "../state/db/BlockAndLogStreamerListener";
-import { Connector, Callback } from "./connector";
-import { ContractDependenciesEthers } from "contract-dependencies-ethers";
-import { Controller } from "../state/Controller";
-import { EthersProvider } from "@augurproject/ethersjs-provider";
-import { EventLogDBRouter } from "../state//db/EventLogDBRouter";
-import { JsonRpcProvider } from "ethers/providers";
-import { PouchDBFactory } from "../state//db/AbstractDB";
-import { SubscriptionEventNames } from "../constants";
-import { UploadBlockNumbers, Addresses } from "@augurproject/artifacts";
-
-const settings = require("@augurproject/sdk/src/state/settings.json");
+import {API} from "../state/getter/API";
```

Comment:
I think your editor is auto formatting for no spaces in the import lists but others (mine included) are doing the opposite. We should sort that out some day.